src/main/java/com/google/maps/GeocodingApiRequest.java

```diff
@@ -20,13 +20,13 @@
 import com.google.maps.internal.ApiConfig;
 import com.google.maps.model.AddressType;
 import com.google.maps.model.ComponentFilter;
-import com.google.maps.model.GeocodingResult;
+import com.google.maps.model.GeocodingResponse;
 import com.google.maps.model.LatLng;
 import com.google.maps.model.LocationType;
 
 /** A request for the Geocoding API. */
```

Comment:
Let's improve upon this javadoc since we're changing GeocodingApiRequest.

Reply:
I added Javadoc comments to all the new fields.